src/smdba/postgresqlgate.py

```diff
@@ -137,6 +137,9 @@ def estimate(self):
         """
         Estimate the data.
         """
+
+        PG_VERSION = "/etc/alternatives/postgresql"
+
```

Comment:
What is this? And why not just _asking_ PostgreSQL which version is running at the moment?:
```python
PG_VERSION = [int(v_el) for v_el in os.popen("psql --version | sed -e 's/.*\s//g'").read().split('.')]
```
On current HEAD it will be: `[9, 4, 12]`, for example.